lib/assets/stylesheets/overrides.scss

```diff
@@ -142,6 +142,14 @@ span.sublist {
   margin-right: 25px;
 }
 
+ul {
```

Comment:
This might break other places using ul purely. I would be more inclined to use an specific selector instead.